Client/Frontend/Browser/Playlist/Controllers/PlaylistCarplayController.swift

```diff
@@ -482,7 +482,7 @@ extension PlaylistCarplayController {
             }
             
             // Attempt to play the stream
-            if let url = URL(string: item.pageSrc) {
+            if let url = URL(string: item.src) {
```

Comment:
So the `pageSrc` might be different and it will not load  playlist item ?

Reply:
It won't be. The `item.src` is guaranteed to be a direct link to the video. The `pageSrc` would be something like `youtube.com/watch?v=VideoID`. In the case where it can't get the video direct link, it will fallback to the webView which will load `pageSrc`.